pom.xml

```diff
@@ -118,58 +118,51 @@
       <subscribe>dev-subscribe@commons.apache.org</subscribe>
       <unsubscribe>dev-unsubscribe@commons.apache.org</unsubscribe>
       <post>dev@commons.apache.org</post>
-      <archive>http://mail-archives.apache.org/mod_mbox/commons-dev/</archive>
+      <archive>https://mail-archives.apache.org/mod_mbox/commons-dev/</archive>
       <otherArchives>
-        <otherArchive>http://markmail.org/list/org.apache.commons.dev/</otherArchive>
-        <otherArchive>http://old.nabble.com/Commons---Dev-f317.html</otherArchive>
-        <otherArchive>http://www.mail-archive.com/dev@commons.apache.org/</otherArchive>
-        <otherArchive>http://news.gmane.org/gmane.comp.jakarta.commons.devel</otherArchive>
+        <otherArchive>https://markmail.org/list/org.apache.commons.dev/</otherArchive>
+        <otherArchive>https://www.mail-archive.com/dev@commons.apache.org/</otherArchive>
       </otherArchives>
     </mailingList>
     <mailingList>
       <name>Commons Issues List</name>
       <subscribe>issues-subscribe@commons.apache.org</subscribe>
       <unsubscribe>issues-unsubscribe@commons.apache.org</unsubscribe>
-      <archive>http://mail-archives.apache.org/mod_mbox/commons-issues/</archive>
+      <archive>https://mail-archives.apache.org/mod_mbox/commons-issues/</archive>
       <otherArchives>
-        <otherArchive>http://markmail.org/list/org.apache.commons.issues/</otherArchive>
-        <otherArchive>http://old.nabble.com/Commons---Issues-f25499.html</otherArchive>
-        <otherArchive>http://www.mail-archive.com/issues@commons.apache.org/</otherArchive>
+        <otherArchive>https://markmail.org/list/org.apache.commons.issues/</otherArchive>
+        <otherArchive>https://www.mail-archive.com/issues@commons.apache.org/</otherArchive>
       </otherArchives>
     </mailingList>
     <mailingList>
       <name>Commons Commits List</name>
       <subscribe>commits-subscribe@commons.apache.org</subscribe>
       <unsubscribe>commits-unsubscribe@commons.apache.org</unsubscribe>
-      <archive>http://mail-archives.apache.org/mod_mbox/commons-commits/</archive>
+      <archive>https://mail-archives.apache.org/mod_mbox/commons-commits/</archive>
       <otherArchives>
-        <otherArchive>http://markmail.org/list/org.apache.commons.commits/</otherArchive>
-        <otherArchive>http://www.mail-archive.com/commits@commons.apache.org/</otherArchive>
+        <otherArchive>https://markmail.org/list/org.apache.commons.commits/</otherArchive>
+        <otherArchive>https://www.mail-archive.com/commits@commons.apache.org/</otherArchive>
       </otherArchives>
     </mailingList>
     <mailingList>
       <name>Apache Announce List</name>
       <subscribe>announce-subscribe@apache.org</subscribe>
       <unsubscribe>announce-unsubscribe@apache.org</unsubscribe>
-      <archive>http://mail-archives.apache.org/mod_mbox/www-announce/</archive>
+      <archive>https://mail-archives.apache.org/mod_mbox/www-announce/</archive>
```

Comment:
Same here https://lists.apache.org/list.html?dev@commons.apache.org ?

Reply:
`dev@` isn't the same as `announce@` but I get the point.

Reply:
Sorry, I meant `dev@`, but I put this comment in wrong place.